### DISCUSSION TITLE
Incompatible Servers

### DISCUSSION THREAD

**Discussion by ghost** [redeclipse/base#1205] [Help and Support] [answered]
So i was playing RE and then i was gonna play a time trial but theres no servers for it i checked incompatible and its all there how do i fix this?

**Comment by SniperGoth** [comment]
Depends, which version you are playing?

**Comment by ghost** [comment]
> Depends, which version you are playing?

2.0.0
I tried 1.6.0 but it doesnt seem to launch
But it says 2.0.1 on some servers but how?

**Comment by Calinou** [comment]
> But it says 2.0.1 on some servers but how?

This is because they're running a development version of the game. If you want to play it, you have to compile it yourself.

If there are no time trial servers on 2.0.0, maybe I should convert mine to be a 24/7 time-trial server...

**Comment by q009** [comment, marked as answer]
> This is because they're running a development version of the game. If you want to play it, you have to compile it yourself.

You actually don't have to, if you have the game on Steam you can just switch over to the development branch in the game properties, in the beta tab.
